frontend/src/components/AboutPage/AboutHeader/ThemeInput/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { CustomInput } from '../../../Inputs';
+
+interface ThemeInputProps {
+    name: string;
+    setName: (s: string) => void;
+    saveState: () => void;
+}
+
+/** Component exposing a textfield for editing a panel category */
+export const ThemeInput: React.FC<ThemeInputProps> = ({
+    name,
+    setName,
+    saveState,
+}) => {
+    return (
+        <div
+            className="w-full flex flex-col min-h-fit justify-start items-center"
+            data-testid="theme-input"
+        >
+            <CustomInput
+                type={'text'}
+                value={name}
+                label={'Theme'}
+                textHelper={'Type the theme of your project'}
+                onInput={({ target }) =>
+                    setName((target as HTMLInputElement).value)
+                }
+                onFocusOut={saveState}
```

Comment:
Now it saves even if user does not write anything.
SaveState method should be improved.
![z3II9ED6VW](https://user-images.githubusercontent.com/30553164/220870983-13d2cb2b-2280-4b8c-811b-011414509880.gif)

Reply:
Improved saving to no longer autosave for unchanged input.